force-app/main/default/classes/ALLO_Allocations_TDTM.cls

```diff
@@ -1176,4 +1194,57 @@ public class ALLO_Allocations_TDTM extends TDTM_Runnable {
         return (val != null) ? val : defaultVal;
     }
 
+    /**
+     * @description Recreates the Scheduled Payment Allocations for Opportunities whose Allocations were updated as a
+     *              result of A Recurring Donations GAU Allocations being updated.
+     */
+    @testVisible private void recreateScheduledPaymentAllocationsIfNecessary() {
+        if (!isPaymentAllocationsEnabled() || updatedRecurringDonationOppIds.isEmpty()) {
+            return;
+        }
+
+        dmlWrapper = (dmlWrapper == null) ? new DMLWrapper() : dmlWrapper;
```

Comment:
Not a big deal. 
Do we need null check here? Since the previous step is already set dmlWrapper to null.

Reply:
I debated, and decided to add it mostly so if it was not null in the future for some reason it would handle it.  I could just allocate it.  Actually, what I will likely do is not reuse the class one.  one that is local to that method is safest.